features/networking/multus.feature

```diff
@@ -866,3 +866,132 @@ Feature: Multus-CNI related scenarios
+    And I run commands on the host:
+      | ip link add mvlanp0 type vxlan id 100 remote <%= cb.mastr_inf_address %> dev <%= cb.workr_inf_name %> dstport 14789 |
+      | ip link set up mvlanp0                                                                                              |
+      | ip a add 192.168.1.2/24 dev mvlanp0                                                                                 |
+    Then the step should succeed
+    #Cleanup for deleting worker interface
+    Given I register clean-up steps:
+    """
+    the bridge interface named "mvlanp0" is deleted from the "<%= cb.worker[0].name %>" node
+    """
+    #Configuing tunnel interface on master node
+    Given I use the "<%= cb.master[0].name %>" node
+    And I run commands on the host:
+      | ip link add mvlanp0 type vxlan id 100 remote <%= cb.workr_inf_address %> dev <%= cb.mastr_inf_name %> dstport 14789 |
+      | ip link set up mvlanp0                                                                                              |
+      | ip a add 192.168.1.1/24 dev mvlanp0                                                                                 |
+    Then the step should succeed
+    #Cleanup for deleting master interface
+    Given I register clean-up steps:
+    """
+    the bridge interface named "mvlanp0" is deleted from the "<%= cb.master[0].name %>" node
+    """
+    #Confirm the link connectivity between master and worker
+    When I run commands on the host:
+      | ping -c1 -W2 192.168.1.2 |
+    Then the step should succeed
+    Given I use the "<%= cb.worker[0].name %>" node
+    And I run commands on the host:
+      | ping -c1 -W2 192.168.1.1 |
+    Then the step should succeed
+    
+    #Configuring DHCP service on master node
+    Given a DHCP service for macvlan tunnel mode is configured for interface "mvlanp0" on "<%= cb.master[0].name %>" node with address range and lease time as "192.168.1.100,192.168.1.120,24h"
+    #Cleanup for deconfiguring DHCP service on target node
+    Given I register clean-up steps:
+    """
+    a DHCP service is deconfigured on the "<%= cb.master[0].name %>" node
+    """
+    #Patching simplemacvlan config in network operator config CRD
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with:
+      | {"spec": {"additionalNetworks": [{"name": "testmacvlan","namespace": "openshift-multus","simpleMacvlanConfig": {"ipamConfig": {"type": "dhcp"},"master": "mvlanp0"},"type": "SimpleMacvlan"}]}} |
+    #Cleanup for bringing CRD to original
+    Given I register clean-up steps:
+    """
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
+    | {"spec":{"additionalNetworks": null}} |
+    """
+    And admin ensures "testmacvlan" network_attachment_definition is deleted from the "openshift-multus" project after scenario
+    
+    #Creating pod under openshift-multus project to absorb above net-attach-def
```

Comment:
can we update the namespace from 'openshift-multus' to 'default' in step 975 .  then here as normal user can consume it.

Reply:
@zhaozhanqi This has to be in openshift-multus project as per test case steps and believe its macvlan tunnel restrictions to configure in openshift-multus namespace only.

Reply:
Actually, we can do it and it should work. Let me change step and try

Reply:
@zhaozhanqi PTAL. Changed it to normal user. Removed admin ensure net-attach-def deletion step step etc and shortened some stuff. Thanks. Re-ran logs http://pastebin.test.redhat.com/824075

Reply:
yeah, great thanks the update
/lgtm